activities/TamTamMicro.activity/app.js

```diff
@@ -105,7 +105,15 @@ enyo.kind({
 	},
 	
 	updateItem: function(sender, event) {
-		this.openItems[event.toChange].deselect();
+		try {
```

Comment:
Wooaw, using try/catch to avoid the issue is not a good practice 😮 
Will be better to test if the `event.toChange` and `this.openItems[event.toChange]` is not null before calling `deselect` method